### ISSUE TITLE
Link to simulator unreachable

### ISSUE THREAD

**Issue by daixtrose** [kisvegabor/lvgl#5137]
### Perform all steps below and tick them with [x]

- [x] Read the [FAQ](https://docs.lvgl.io/master/intro/index.html#faq)
- [x] Check the related part of the [Documentation](https://docs.lvgl.io/)
- [x] Update lvgl to the latest version
- [ ] ~~Reproduce the issue in a [Simulator](https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html)~~

### Describe the bug

In the [README.md](https://github.com/lvgl/lvgl/blob/master/README.md#arrow_forward-get-started) in the `Getting Started` section the subsection `Start to Use LVGL` the link in 

```ghfm
    Set up a [Simulator](https://docs.lvgl.io/master/get-started/platforms/pc-simulator.html) (10 minutes)
```

leads to a 404 message. 

### To Reproduce

Visit https://docs.lvgl.io/master/get-started/platforms/pc-simulator.html, which yields a 404.

### Expected behavior

A webpage to read stuff.

### Screenshots or video

![grafik](https://github.com/lvgl/lvgl/assets/5588692/16d32c46-c825-44f3-987d-c37837ce6efe)

**Comment by daixtrose** [comment]
AFAICS via digging through the docs, the link should be https://docs.lvgl.io/master/integration/ide/pc-simulator.html#simulator

**Comment by C47D** [comment]
Hi, I've submitted a fix in #5139

**Comment by kisvegabor** [comment]
Thanks! Let's continue there!
